mdd.py

```diff
@@ -278,7 +293,10 @@ def get_cpu_info():
         inxi_info = get_inxi_main_cat("#CPU")
 
         for item in inxi_info:
-            val = get_inxi_val(item, "#model")
+            if aarch64:
+                val = get_inxi_val(item, "#variant")
```

Comment:
Some ARM devices have multiple variants: `Info: quad core (2-mt/2-st) model: N/A variant-1: cortex-a72 variant-2: cortex-a53 bits: 64`. Here we just check for the value `variant`

Reply:
Changed this code. Needs test on an ARM chip with two variants.

Reply:
Tested by @Dark-Sky